src/modules/fundingManager/depositVault/interfaces/IFM_DepositVault_v1.sol

```diff
@@ -10,6 +17,14 @@ interface IFM_DepositVault_v1 {
     /// @param  _amount The amount of tokens deposited.
     event Deposit(address indexed _from, uint _amount);
 
+    /// @notice Event emitted when protocol fee has been transferred to the treasury.
+    /// @param  token The token received as protocol fee.
+    /// @param  treasury The protocol treasury address receiving the token fee amount.
+    /// @param  feeAmount The fee amount transferred to the treasury.
+    event ProtocolFeeTransferred(
```

Comment:
We are currently duplicating this event across differnt interfaces. Might it be an idea to move it to the IFundingManager_v1 interface, or IModule_v1 interface so that all fee collection has access to the same event?